modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -920,6 +920,31 @@ CREATE TEXT INDEX FOR (n:Label) ON (n.prop) OPTIONS {indexProvider : 'text-2.0'}
 ----
 ====
 
+.Using differently ordered return items in a `UNION` clause
+====
+Query::
++
+[source,cypher]
+----
+RETURN 'val' as one, 'val' as two
+UNION
+RETURN 'val' as two, 'val' as one
+----
+
+Description of the returned code::
+All sub queries in a UNION should have the same ordering for the return column names, this is deprecated and will be enforced in later versions.
```

Comment:
- Looking at the Cypher manual, we usually write 'subqueries' as one word
- I think we can talk about just 'return columns' rather than ''return column names' 
- In the other deprecations we talk about 'a future version' rather than 'later versions'
- It sounds a bit like it is the same ordering that is deprecated, so I would reformulate it to be clearer that it is different order that is deprecated. Not sure what the best wording would be (maybe you have a better suggestion @renetapopova or @gem-neo4j ), but I'm thinking something along the lines of

Different ordering of  the return columns in the subqueries in a UNION is deprecated. Same ordering for the return columns will be enforced in a future version.